OracleGoldenGate/21c/README.md

```diff
@@ -41,18 +44,30 @@ To create a container image for GoldenGate for Oracle Database, use the followin
 
 ```sh
 $ docker build --tag=oracle/goldengate:21.3.0.0.0 \
-                --build-arg INSTALLER=213000_fbo_ggs_Linux_x64_Oracle_services_shiphome.zip .
+               --build-arg INSTALLER=213000_fbo_ggs_Linux_x64_Oracle_services_shiphome.zip .
 Sending build context to Docker daemon
 ...
 Successfully tagged oracle/goldengate:21.3.0.0.0
 ```
 Similarly, for other Databases like BigData, MySQL, PostgreSQL, etc. provide the name of the zip file for the INSTALLER argument.
 
+### Changing the Base Image
+
+By default, the base image used by Docker to build Oracle GoldenGate Docker image is `oraclelinux:8`, but this can be changed.
```

Comment:
Please remove "Docker" from this sentence, i.e. "By default, the base image used by the Oracle GoldenGate container image is `oraclelinux:8` but this can be changed."

It would also be good to provide either a list of valid base images or criteria by which someone could determine if something would work as a base image.